src/app/builders/components/Container/MetricsContainer.tsx

```diff
@@ -0,0 +1,19 @@
+import { cn } from '@/lib/utils'
+import { FC, PropsWithChildren } from 'react'
+import { CommonComponentProps } from '../commonProps'
+
+export type MetricsContainerProps = CommonComponentProps & PropsWithChildren
+
+export const MetricsContainer: FC<MetricsContainerProps> = ({ dataTestid, className = '', children }) => {
```

Comment:
What is the difference between this component and the one in `Backers`? If there is none, I would reuse it, remember that we can just call components from parents components.